ruby-challenges.rb

```diff
@@ -33,4 +56,21 @@
 palindrome_test_case3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
 
-# Pseudo code:
\ No newline at end of file
+# Pseudo code:
+# create a function named fwd that takes string as a parameter
+def fwd(string)
+  # if string is equal to reversed string
+  if string. == string.reverse
+    # return string is a palindrome
+    "#{string} is a palindrome"
+  else
+    # else string is not a palindrome
+    "#{string} is not a palindrome"
+  end  
+end   
+# print function fwd with string of invoked variable in all lowercase
+p fwd(palindrome_test_case1.downcase) # "racecar is a palindrome"
+p fwd(palindrome_test_case2.downcase) # "learn is not a palindrome"
+p fwd(palindrome_test_case3.downcase) # "rotator is a palindrome"
+
+  
```

Comment:
Awesome job, Jackson!  One tip - use the .downcase in your method, rather than when calling it.  Just a best practice tip for you on this lovely Tuesday evening.  🧐